http-server/src/test/java/org/triplea/server/lobby/chat/MessagingServiceTest.java

```diff
@@ -46,8 +46,11 @@
 
   private static final ChatParticipantAdapter chatParticipantAdapter = new ChatParticipantAdapter();
 
-  private static final ApiKeyUserData API_KEY_USER_DATA =
-      ApiKeyUserData.builder().role(UserRole.MODERATOR).username("player-name-moderator").build();
+  private static final UserWithRoleRecord API_KEY_USER_DATA =
+      UserWithRoleRecord.builder()
+          .role(UserRole.MODERATOR)
+          .username("player-name-moderator")
+          .build();
```

Comment:
And here as well

Reply:
Calling this `MODERATOR_DATA`, `MODERATOR_RECORD` is misleading, and probably also not the best to just name this data after its type but instead to match what it represents.